src/containers/NoteList.tsx

```diff
@@ -27,7 +27,7 @@ const NoteList: React.FC = () => {
   const filteredNotes: NoteItem[] = notes
     .filter(filter[activeFolder])
     .filter(isMatch)
-    .sort(sortByLastUpdated)
+    .sort(sortByFavouritesThenLastUpdated)
```

Comment:
How about doing it in two sorts?

One sort by last updated and other by favourite. First the sort by date and then by favourite

What do you think?

Reply:
I originally had it as two sorts, however I figured this would be more perfomant as we're only iterating over the array once. However after benchmarking the options, the difference is negligable so I'm not opposed to either way.

Will make the change! 👍